mozperftest_tools/tools/side_by_side.py

```diff
@@ -0,0 +1,865 @@
+        test_no_e10s = test_name.replace("-e10s", "")
+
+        # Download the artifacts
+        if not skip_download:
+            base_paths = []
+            for base_revision_id in base_revision_ids:
+                if base_paths:
+                    break
+                artifact_downloader(
+                    base_revision_id,
+                    output_dir=str(output),
+                    test_suites=[test_name],
+                    platform=platform,
+                    artifact_to_get=["browsertime-results", "perfherder-data"],
+                    unzip_artifact=True,
+                    download_failures=True,
+                    ingest_continue=False,
+                )
+                base_paths = self._search_for_paths([base_revision_id], "browsertime-results")
+                base_vismet = self._search_for_paths([base_revision_id], "perfherder-data")
+
+                # Get the vismet perfherder data now
+                artifact_downloader(
+                    base_revision_id,
+                    output_dir=str(output),
+                    test_suites=[test_no_e10s],
+                    platform=vismet_platform,
+                    artifact_to_get=["perfherder-data"],
+                    unzip_artifact=False,
+                    download_failures=True,
+                    ingest_continue=False,
+                )
+                vismet_paths = self._search_for_paths([base_revision_id], "perfherder-data")
+
+                base_vismet_taskids = []
+                for path in vismet_paths[0]:
+                    base_vismet_taskids.append(os.path.split(path)[1].split("_")[0])
+
+                base_vismet_mapping = match_vismets_with_videos(
+                    base_revision_id, str(output), base_vismet_taskids
+                )
+                base_vismet[0].extend(vismet_paths[0])
+
+            new_paths = []
+            for new_revision_id in new_revision_ids:
+                if new_paths:
+                    break
+                artifact_downloader(
+                    new_revision_id,
+                    output_dir=str(output),
+                    test_suites=[new_test_name or test_name],
+                    platform=new_platform or platform,
+                    artifact_to_get=["browsertime-results", "perfherder-data"],
+                    unzip_artifact=True,
+                    download_failures=True,
+                    ingest_continue=False,
+                )
+                new_paths = self._search_for_paths([new_revision_id], "browsertime-results")
+                new_vismet = self._search_for_paths([new_revision_id], "perfherder-data")
+
+                # Get the vismet perfherder data now
+                artifact_downloader(
+                    new_revision_id,
+                    output_dir=str(output),
+                    test_suites=[test_no_e10s],
+                    platform=vismet_platform,
```

Comment:
Are you using new revisions in your tests? This line is looking for vismet platforms in the given revision. If the revisions are after the rework, this will break.